source/common/config/filesystem_subscription_impl.h

```diff
@@ -33,8 +36,10 @@ class FilesystemSubscriptionImpl : public Config::Subscription<ResourceType> {
     callbacks_ = &callbacks;
     watcher_->addWatch(path_, Filesystem::Watcher::Events::MovedTo, [this](uint32_t events) {
       UNREFERENCED_PARAMETER(events);
-      refresh();
+      refresh(false);
     });
+    // Attempt to read, gracefully fail if the file isn't there yet.
```

Comment:
I was trying to distinguish between an "internal" failure, where the Subscription mechanism fails, e.g. due to a network error etc. and a configuration issue where we reject due to bogus config (e.g. a route table referring to non-existent clusters as per #1199). It seems it might be useful to know at stats level whether failures are due to one or the other.. WDYT? @tschroed as well.